.env

```diff
@@ -0,0 +1,10 @@
+AKENEO_API_BASE_URI=https://my-pim.com
```

Comment:
You should use the bootstrap.php, which load env variables instead of an .env file.
These env variables should already exists if you have a working EE.